Aztec/Classes/Formatters/Implementations/HTMLParagraphFormatter.swift

```diff
@@ -31,6 +31,14 @@ public class HTMLParagraphFormatter: ParagraphAttributeFormatter {
 
         var resultingAttributes = attributes
         resultingAttributes[.paragraphStyle] = newParagraphStyle
+
+        if let font = attributes[.font] as? UIFont,
+           font.familyName != UIFont.systemFont(ofSize: UIFont.systemFontSize).familyName {
```

Comment:
System font's are automatically scaled by default, which led to errors in the automated tests (due to attempts to scale an already scaled font).

Reply:
👋 Thanks for the note @SiobhyB! Were the failing CI tests for this repo or in WPiOS? I ask because I'm seeing failures [in this PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/22242).

Reply:
Thank you for pointing those out. The failing tests I referred to here were in this repo, but I see the ones in WPiOS are relevant too. Apologies for not spotting before requesting review! The tests prompted me to review the approach I took, and I think I have a better approach now. I'll update shortly.

Reply:
I believe this warrants a change in `RCTAztecView.swift` in Gutenberg, rather than here. As such, I've gone ahead to create a new PR at https://github.com/WordPress/gutenberg/pull/57339. I'll close this one and request review on that one when it's done. 🙇‍♀️

Reply:
Oh nice! Thank you @SiobhyB. 🙇